definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-resp-journey-nonprod.json

```diff
@@ -383,5 +383,138 @@
     "CaseFieldID": "RespLegalProceedingsDescription",
     "UserRole": "[RESPSOLICITOR]",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DigitalNoticeOfProceedings",
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DigitalNoticeOfProceedings",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DigitalNoticeOfProceedings",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DigitalNoticeOfProceedings",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DigitalNoticeOfProceedings",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DigitalNoticeOfProceedings",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "NoticeOfProceedingsEmail",
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "NoticeOfProceedingsEmail",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "NoticeOfProceedingsEmail",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "NoticeOfProceedingsEmail",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "NoticeOfProceedingsEmail",
+    "UserRole": "caseworker-divorce-courtadmin",
```

Comment:
I think this one and the legal advisor role (i.e. caseworker-divorce-courtadmin-la) only need "R" permission.

Reply:
I think it's because of:
https://tools.hmcts.net/jira/browse/RDM-11628

CRU for all fields and limit it on event level. At least until CCD provides answers.